src/gtk/app.cpp

```diff
@@ -347,6 +351,36 @@ bool wxApp::SetNativeTheme(const wxString& theme)
 #endif
 }
 
+wxApp::AppearanceResult wxApp::SetAppearance(Appearance appearance)
+{
+#ifdef __WXGTK3__
+    gboolean preferDark = FALSE;
```

Comment:
As far as I know, it's perfectly fine to assign `true`/`false` to an `int`, so I don't see any reason not to prefer those over the old `TRUE`/`FALSE`.

Reply:
IMO this is more clear: even if C++ allows you to assign just about [anything to anything else](https://fosstodon.org/@VZ/112235960416690642), using `false` with `bool` variables and `FALSE` with `int` ones is IMO tidier and I've always done it like this in wxMSW (which has its own `BOOL` which is actually `int`) and would rather keep doing it in wxGTK too (where there are already more than 330 occurrences of these constants) unless you absolutely hate it.